src/cmap/connection.ts

```diff
@@ -95,22 +95,6 @@ export interface CommandOptions extends BSONSerializeOptions {
   writeConcern?: WriteConcern;
 }
 
-/** @internal */
-export interface GetMoreOptions extends CommandOptions {
```

Comment:
This interface was duplicated, caused an issue when merging all the imports to one file, I've de-duped (all internal, no public API impact)